tests/std/tests/P1020R1_smart_pointer_for_overwrite/test.cpp

```diff
@@ -0,0 +1,344 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <cstdint>
+#include <cstdlib>
+#include <cstring>
+#include <memory>
+#include <new>
+#include <stdexcept>
+#include <string_view>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+using namespace std;
+
+constexpr int uninitializedValue = 0xEE;
+constexpr int defaultValue       = 106;

```

Comment:
The meaning of 'defaultValue` is very broad. Perhaps: initializedMemoryValue?

Reply:
Renamed to `initializedValue`